client/dogs/SearchPage.js

```diff
@@ -43,40 +37,35 @@ export default class IndexPage extends React.Component {
     const { query } = ctx;
 
     let results = null;
+    //console.log('QUERY', query);
 
-    if (query.q) {
+    if (query.firstName && query.lastName && query.dogName && query.year) {
```

Comment:
Ah- I haven't moved the search results to the form page yet. Wanted to make the dependencies commit first and then add all the graphql params and move the function over. Should I still remove this code?